packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/xcpd-request.ts

```diff
@@ -38,7 +38,8 @@ export function transformIti55RequestToPatientResource(
     value: id._extension,
   }));
 
-  const gender = mapIheGenderToFhir(queryParams.livingSubjectAdministrativeGender?.value?._code);
+  const iheGender = queryParams.livingSubjectAdministrativeGender?.value;
+  const gender = iheGender ? mapIheGenderToFhir(iheGender._code) : mapIheGenderToFhir(undefined);
```

Comment:
nit: can't this be simplified to `const gender = mapIheGenderToFhir(iheGender?._code)`